scrapy/logformatter.py

```diff
@@ -13,25 +13,29 @@
 class LogFormatter(object):
     """Class for generating log messages for different actions.
 
-    All methods must return a dictionary listing the parameters `level`, `msg`
-    and `args` which are going to be used for constructing the log message when
-    calling logging.log.
+    All methods must return a dictionary listing the parameters ``level``, ``msg``
+    and ``args`` which are going to be used for constructing the log message when
+    calling ``logging.log``.
 
     Dictionary keys for the method outputs:
-        * `level` should be the log level for that action, you can use those
-        from the python logging library: logging.DEBUG, logging.INFO,
-        logging.WARNING, logging.ERROR and logging.CRITICAL.
 
-        * `msg` should be a string that can contain different formatting
-        placeholders. This string, formatted with the provided `args`, is going
-        to be the log message for that action.
+        *   ``level`` is the log level for that action, you can use those from the 
+            `python logging library <https://docs.python.org/3/library/logging.html>`_ :
+            ``logging.DEBUG``, ``logging.INFO``, ``logging.WARNING``, ``logging.ERROR``
+            and ``logging.CRITICAL``.
+
+        *   ``msg`` should be a string that can contain different formatting placeholders. This string, formatted
+            with the provided ``args``, is going to be the long message for that action.
+
+        *   ``args`` should be a tuple or dict with the formatting placeholders for ``msg``. The final log message is
+            computed as ``msg % args``.
 
-        * `args` should be a tuple or dict with the formatting placeholders for
-        `msg`.  The final log message is computed as output['msg'] %
-        output['args'].
     """
```

Comment:
Here I am missing a simple example. Examples are a really powerful tool; people very familiar with Scrapy may see the example and the setting name, and already know what to do, without actually reading the remaining text.

You could feature something in the lines of the [example from Stackoverflow](https://stackoverflow.com/a/34320948/939364), as we know for sure that some users will find it useful.